marketplace.xml

```diff
@@ -11340,4 +11340,43 @@ Capabilities:
     </versions>
   </market_entry>
 
+  <market_entry>
+    <id>pentaho-cde-atolcd-plugins</id>
```

Comment:
Your plugin id must match the name of the folder where the plugin is installed. If unzipping the zip means that you get a folder called atolcd, then that should be your plugin id. Or unzipping the zip should produce a folder called pentaho-cde-atolcd-plugins.

thanks!